Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM openjdk:17-jdk-slim
+
+EXPOSE 19132/tcp
+EXPOSE 19132/udp
+
+WORKDIR /home
+
+ADD target/Waterdog.jar /home
+
+ENTRYPOINT ["java", "-jar", "Waterdog.jar"]
```

Comment:
can we make this configurable so that people can set their own JVM flags and memory option?

Reply:
The ENTRYPOINT is just the default command to launch the process. You can individually overwrite it in the docker run command if you want to do that. Same thing goes for k8s or other such solutions.